dataRetriever/mock/intRandomizerMock.go

```diff
@@ -2,12 +2,16 @@ package mock
 
 // IntRandomizerMock -
 type IntRandomizerMock struct {
-	IntnCalled func(n int) (int, error)
+	IntnCalled func(n int) int
 }
 
 // Intn -
-func (irm *IntRandomizerMock) Intn(n int) (int, error) {
-	return irm.IntnCalled(n)
+func (irm *IntRandomizerMock) Intn(n int) int {
+	if irm.IntnCalled != nil {
+		return irm.IntnCalled(n)
+	}
+
+	return 0
```

Comment:
maybe return 1 in case we want to use this when dividing by something. Also this seems to be a stub, not a mock

Reply:
renamed to stub. Regarding the returned value, might keep 0 as any randomizer can (and at some point in time will) return 0. If someone carelessly use this as a divider, will have a surprise :)